bot/cogs/help_channels.py

```diff
@@ -544,14 +550,53 @@ def is_dormant_message(message: t.Optional[discord.Message]) -> bool:
         # be put in the queue.
         await self.move_to_available()
 
+    @staticmethod
+    async def ensure_permissions_synchronization(category: discord.CategoryChannel) -> None:
+        """
+        Ensure that all channels in the `category` have their permissions synchronized.
+
+        This method mitigates an issue we have yet to find the cause for: Every so often, a channel in the
+        `Help: Available` category gets in a state in which it will no longer synchronizes its permissions
+        with the category. To prevent that, we iterate over the channels in the category and edit the channels
+        that are observed to be in such a state. If no "out of sync" channels are observed, this method will
+        not make API calls and should be fairly inexpensive to run.
+        """
+        for channel in category.channels:
```

Comment:
I haven't benchmarked but I try to avoid that attribute because it sorts. That's why I made `self.get_category_channels()`.